tcms/core/tests/test_middleware.py

```diff
@@ -14,16 +14,21 @@
     "CheckUnappliedMigrationsMiddleware testing not enabled",
 )
 class TestCheckUnappliedMigrationsMiddleware(test.TransactionTestCase):
```

Comment:
Must be an instance of LoggedInTestCase IMO, otherwise `/` will be redirected to /accounts/login.